src/network/bridge.rs

```diff
@@ -69,6 +70,17 @@ impl<'a> Bridge<'a> {
     }
 }
 
+fn get_mac_address(v: Vec<Nla>) -> NetavarkResult<String> {
```

Comment:
move this function to core_utils.rs then make all coders import it and use it from there